/**:
  controller_manager:
    ros__parameters:
      update_rate: 500  # Hz
      moveit_controller_manager: moveit_ros_control_interface/MoveItControllerManager

      moveit_ros_control_interface:
        controller_names:
          - ur_left_joint_group_vel_controller
          - ur_right_joint_group_vel_controller
          - srm_r_base_controller
          - srm_l_base_controller

      ur_left_joint_group_vel_controller:
        type: velocity_controllers/JointGroupVelocityController
        action_ns: /follow_joint_trajectory
        joints:
          - ur_left_shoulder_pan_joint
          - ur_left_shoulder_lift_joint
          - ur_left_elbow_joint
          - ur_left_wrist_1_joint
          - ur_left_wrist_2_joint
          - ur_left_wrist_3_joint

        pid_gains:
          ur_left_shoulder_pan_joint: {p: 10.0, i: 0.05, d: 0.1}
          ur_left_shoulder_lift_joint: {p: 10.0, i: 0.05, d: 0.1}
          ur_left_elbow_joint: {p: 10.0, i: 0.05, d: 0.1}
          ur_left_wrist_1_joint: {p: 10.0, i: 0.05, d: 0.1}
          ur_left_wrist_2_joint: {p: 10.0, i: 0.05, d: 0.1}
          ur_left_wrist_3_joint: {p: 10.0, i: 0.05, d: 0.1}

        interface_name: velocity
      
      ur_right_joint_group_vel_controller:
        type: velocity_controllers/JointGroupVelocityController
        action_ns: /follow_joint_trajectory
        joints:
          - ur_right_shoulder_pan_joint
          - ur_right_shoulder_lift_joint
          - ur_right_elbow_joint
          - ur_right_wrist_1_joint
          - ur_right_wrist_2_joint
          - ur_right_wrist_3_joint

        pid_gains:
          ur_right_shoulder_pan_joint: {p: 10.0, i: 0.05, d: 0.1}
          ur_right_shoulder_lift_joint: {p: 10.0, i: 0.05, d: 0.1}
          ur_right_elbow_joint: {p: 10.0, i: 0.05, d: 0.1}
          ur_right_wrist_1_joint: {p: 10.0, i: 0.05, d: 0.1}
          ur_right_wrist_2_joint: {p: 10.0, i: 0.05, d: 0.1}
          ur_right_wrist_3_joint: {p: 10.0, i: 0.05, d: 0.1}

        interface_name: velocity

      srm_r_base_controller:
        type: FollowJointTrajectory
        action_ns: /follow_joint_trajectory
        joints:
          - right_summit_back_left_wheel_joint
          - right_summit_back_right_wheel_joint
          - right_summit_front_left_wheel_joint
          - right_summit_front_right_wheel_joint
      
      srm_l_base_controller:
        type: FollowJointTrajectory
        action_ns: /follow_joint_trajectory
        joints:
          - left_summit_back_left_wheel_joint
          - left_summit_back_right_wheel_joint
          - left_summit_front_left_wheel_joint
          - left_summit_front_right_wheel_joint
  


  # joint_state_broadcaster:
  #   ros__parameters:
  #     use_local_topics: true    


